protecode/client.py

```diff
@@ -138,14 +153,22 @@ def _request(self, method, *args, **kwargs):
         else:
             cookies = None
 
-        auth = self._auth
+        method_kwargs = {
+            'verify': self._tls_verify,
```

Comment:
if I see this properly, `verify`, `cookies`, and `headers` are always passed, and the only dynamic kwarg would be `auth`.

since this pattern of passing dynamic kwargs decreases readability, please do not do this unless actually required.

note that it is also not necessarily required to manually set the `Authorization`-header; requests also supports this (I think). But it is also ok for me to do this manually